src/main/java/lotto/domain/LottoStatistic.java

```diff
@@ -0,0 +1,61 @@
+package lotto.domain;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class LottoStatistic {
+    private static final String RANK_FOUR_REWARD_MESSAGE = "3개 일치 (5000원)- %d개\n";
+    private static final String RANK_THIRD_REWARD_MESSAGE = "4개 일치 (50000원)- %d개\n";
+    private static final String RANK_SECOND_REWARD_MESSAGE = "5개 일치 (1500000원)- %d개\n";
+    private static final String RANK_FIRST_REWARD_MESSAGE = "6개 일치 (2000000000원)- %d개\n";
+    private static final String PROFIT_RATE_MESSAGE = "총 수익률은 %.2f입니다.(기준이 1이기 때문에 결과적으로 손해라는 의미임)";
+    private static final int PURCHASE_UNIT = 1000;
+
+    private int lottCount;
+    private Map<Rank, Integer> rankCount = new HashMap<>();
+
+    public LottoStatistic(List<Integer> matchCounts, LottoCount lottoCount) {
+        lottCount = lottoCount.count();
+        setRankCount(matchCounts);
+    }
+
+    public void setRankCount(List<Integer> matchCounts) {
+        for (int count : matchCounts) {
+            Rank rank = Rank.valueOfRank(count);
+            if (rankCount.get(rank) == null) {
+                rankCount.put(rank, 1);
+                continue;
+            }
+            rankCount.put(rank, rankCount.get(rank) + 1);
```

Comment:
map에서 제공하는 메서드는 생각보다 많기 때문에 이를 활용해도 좋을 것 같아요!

```suggestion
            rankCount.put(rank, rankCount.getOrDefault(rank, 0) + 1);
```

Reply:
꿀팁 감사합니다!! 적극 활용하겠습니다.